docs/schema.rst

```diff
@@ -25,7 +25,9 @@ and machine-readable, and to inter-operate with standards being developed in org
 Overview
 ~~~~~~~~
 
-The diagram below shows an overview of the schema elements.
+The diagram below shows an overview of Phenopackets schema elements and relationships. The orange circles represent an Interpretation and its sub-elements. Blue circles are the top-level elements used to describe an individual or group of individuals, and the green circles are the building block components. Lines between elements indicate composition.
```

Comment:
Please update this using the https://raw.githubusercontent.com/phenopackets/phenopacket-schema/v1.1/docs/schema.rst to work off. Sorry, we slightly deviated on that one!

Reply:
@julesjacobsen Updated - please double-check that it's what you were looking for!